_posts/2016/2016-04-25-hackathon-2016.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "ePages Hackathon 2016"
+date: "2016-04-25 07:16:17"
+icon: code
+categories: events
+authors: ["Birgit"]
+---
+
+Another year, another ePages hackathon.
+Spring is a nice season for hackdays, isn't it?
+The world comes alive, the days are longer, and our developers have even more energy... for coding {% emoji wink %}!
+But what exactly did we do?
+
+## Ready, steady...
+
+{% image blog/blog-hack16-prep.jpg 50% right %} {% endimage %}
+
+This year's hackathon should focus on app development.
+Now that this was clear, we started to collect more detailed topics and built individual teams.
+The Scrum Masters addressed all manner of things associated with the hackathon organisation:
+Meeting rooms had to be booked, working spaces to be coordinated, snacks and beverages to be organised, as well as the final topics had to be agreed.
+
+And finally, it was time for the colleagues from Jena and Barcelona to travel to ePages headquarter in Hamburg.
+Prior to the event itself, some developers met in bars or clubs in Hamburg to outline their ideas, or prepare their GitHub repositories.
+
+## ...go!
+
+{% image blog/blog-hack16-agenda.jpg 50% left %} {% endimage %}
+
+At 9.00 a.m. sharp our CTO Harm Behrens kicked off the second ePages hackathon.
+Once the agenda was presented, the developers slit up into their teams, settled down and got hacking.
+The next 36 hours then basically consisted of three main activities: hacking, teamwork, and fun.
+
+Well, and also the challenge of feeding developers glued to their computers isn’t a small one.
+Thankfully, tons of crisps, and cookies as well as pizza and lemonade helped to focus on the tough tasks they came to solve, while still having the energy to take breaks and clear their heads.
+
+## Ship it!
+
+{% image blog/blog-hack16-result.jpg %} {% endimage %}
+
+After much weaving of code and pushing of pixels, and simply having a great time together, the teams presented their projects.
```

Comment:
, and simply having a great time together - > while having a great time together